meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb

```diff
@@ -41,6 +44,15 @@ RDEPENDS:${PN}-tools-base = " \
 
 RDEPENDS:${PN}-tools-dts = " \
     tpm2-tools \
+    packagegroup-tpm2 \
```

Comment:
@PLangowski are those two packagegroup (tpm2 and tpm2-initramfs) needed? What they provide?

Reply:
After looking into the source layer I remove unnecessary duplicates 69264d7c54955ac7aaaed830898fb1f3b0ec193d